amf2/amf_init.sh

```
@@ -0,0 +1,15 @@
+
+cp /mnt/amf/amf.yaml install/etc/open5gs
+sed -i 's|AMF_IP|'$AMF_IP'|g' install/etc/open5gs/amf.yaml
+sed -i 's|SCP_IP|'$SCP_IP'|g' install/etc/open5gs/amf.yaml
+sed -i 's|NRF_IP|'$NRF_IP'|g' install/etc/open5gs/amf.yaml
+sed -i 's|MNC|'$MNC'|g' install/etc/open5gs/amf.yaml
+sed -i 's|MCC|'$MCC'|g' install/etc/open5gs/amf.yaml
+sed -i 's|TAC|'$TAC'|g' install/etc/open5gs/amf.yaml
+sed -i 's|MAX_NUM_UE|'$MAX_NUM_UE'|g' install/etc/open5gs/amf.yaml
+
+cd install/bin
+exec ./open5gs-amfd $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)